css/styles.css

```diff
@@ -0,0 +1,288 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #f4f5f7;
+}
+
+header {
+  background: #f72210;
+}
+
+img {
+  width: 150px;
+}
+
+.cloud {
+  font-size: 16px;
+}
+
+.subscribe {
+  padding: 5px;
+  vertical-align: middle;
+}
+
+.nav-brand-big {
+  width: 350px;
+}
+
+.bar {
+  width: 10px;
+}
+
+.search-form {
+  border: none;
+  width: 85px;
+  height: 20px;
+  margin-top: 15px;
+}
+
+#extention::before {
+  width: 70px;
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: "";
+  display: block;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+li {
+  list-style: none;
+}
+
+nav li {
+  height: 60px;
+  -webkit-flex-grow: 1;
+  flex-grow: 1;
+  text-align: center;
+  margin-left: -10px;
+}
```

Comment:
The width of these elements is not the same as the original page, that's why your search bar is hidden. Please make the necessary changes to achieve the original look of the page. Your page:
![Screenshot from 2020-03-14 18-17-16](https://user-images.githubusercontent.com/51926937/76692738-107f8280-6620-11ea-96e0-57a9c4be4b5d.png)

Original:
![Screenshot from 2020-03-14 18-17-11](https://user-images.githubusercontent.com/51926937/76692742-14130980-6620-11ea-8768-60594f5fe8f8.png)